src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -1,16 +1,16 @@
-import React from 'react';
-import Header from '../../components/Header/Header';
-import Nav from '../../components/Nav/Nav';
-import ToDo from '../../components/ToDo/ToDo';
-
-const HomeScreen = () => {
-    return(
-        <>
-        <Header />
-        <Nav />
-        <ToDo />
-        </>
-    );
-};
-
-export default HomeScreen;
+import React from "react";
+import Header from "../../components/Header/Header";
+import Nav from "../../components/Nav/Nav";
+import ToDo from "../../components/ToDo/ToDo";
+
+const HomeScreen = () => {
+  return (
+    <>
+      <Header />
+      <Nav />
+      <ToDo />
+    </>
```

Comment:
You made LayoutWrapper to wrap whole each screen, so you should return LayoutWrapper instead React Fragments.